tests/unit/admin/test-class-wc-rest-payments-settings-controller.php

```diff
@@ -451,22 +451,14 @@ public function test_upe_update_settings_saves_enabled_payment_methods() {
 	}
 
 	public function test_upe_split_update_settings_saves_enabled_payment_methods() {
-			$this->mock_upe_split_payment_gateway->update_option( 'upe_enabled_payment_method_ids', [ Payment_Method::CARD ] );
+		$this->mock_split_upe_payment_gateway->update_option( 'upe_enabled_payment_method_ids', [ Payment_Method::CARD ] );
 
-			$request = new WP_REST_Request();
-			$request->set_param( 'enabled_payment_method_ids', [ Payment_Method::CARD, Payment_Method::GIROPAY ] );
-
-			$this->upe_split_controller->update_settings( $request );
-
-			$this->assertEquals( [ Payment_Method::CARD, Payment_Method::GIROPAY ], $this->mock_upe_split_payment_gateway->get_option( 'upe_enabled_payment_method_ids' ) );
-	}
+		$request = new WP_REST_Request();
+		$request->set_param( 'enabled_payment_method_ids', [ Payment_Method::CARD, Payment_Method::GIROPAY ] );
 
-	public function test_update_settings_validation_fails_if_invalid_gateway_id_supplied() {
```

Comment:
This test was removed because it's an integration test that works only with the legacy gateway. It's not easy nor elegant to mock gateway in `WC_Payments` because methods/objects are static. 

Since the tested method relies solely on [`get_upe_available_payment_methods`](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/admin/class-wc-rest-payments-settings-controller.php#L94C40-L94C73), `get_upe_available_payment_methods` is now fully tested by `test-class-upe-payment-gateway.php::test_get_upe_available_payment_methods()`